### ISSUE TITLE
Minimum deployment target

### ISSUE THREAD

**Issue by Natelegreat1** [exyte/fan-menu#39]
Hello.
The README says min deployment target is iOS 8.0
My project's min deployment target is 9.0, yet I am getting this error using cocoa pods:
 
```
[!] CocoaPods could not find compatible versions for pod "FanMenu":
  In Podfile:
    FanMenu (from `https://github.com/exyte/fan-menu`)

Specs satisfying the `FanMenu (from `https://github.com/exyte/fan-menu`)` dependency were found, but they required a higher minimum deployment target.
```

**Comment by kevintbradbury** [comment]
@Natelegreat1 I believe you can change your build settings to Legacy and that should solve the Pods issue, however Macaw was updated for Swift 5 and since the update, I haven't been able to initialize the FanMenu properly.

**Comment by amarunko** [comment]
s.platform     = :ios, '9.1' in podspec

Minimum deployment target of Fan Menu - 9.1

**Comment by amarunko** [comment]
Sorry fo the inconvenience, we will fix README

**Comment by ystrot** [comment]
Moved it back to 9.0, since there is no reason for 9.1. This fix available in the latest 0.7.3 version.
